src/test/test_http_client_builder.rs

```diff
@@ -0,0 +1,12 @@
+use reqwest::{
+    header::{self, HeaderMap, HeaderValue},
+    ClientBuilder,
+};
+
+/// Create a http client builder that with some default headers
+pub fn test_http_client() -> ClientBuilder {
```

Comment:
The HTTP client is unrelated to the data structures for tests, so I would move this to `src/http_client.rs`. And I think we can rename the function to `http_client`, or `custom_http_client` if we want to be more explicit.